src/applications/hca/components/CustomReviewField.jsx

```diff
@@ -11,4 +12,8 @@ const CustomReviewField = ({
   </div>
 );
 
+CustomReviewField.propTypes = {
+  children: PropTypes.object,
```

Comment:
@Matth-Ng just an observation, I was wondering about PropTypes.object for children. It definitely makes sense as it is an object and in this case used to get properties from the object. In react Proptypes documentation I see it set to PropTypes.element or PropTypes.node.